src/generator/__snapshots__/errata.integration.js.snap

```diff
@@ -60,7 +61,7 @@ exports[`aggregates multiple errata into a single play 1`] = `
     - when: check_out.rc == 100
       name: set reboot fact
       set_fact:
-        insights_needs_reboot: true
+        insights_needs_reboot: \\"{{requires_reboot}}\\"
```

Comment:
@semtexzv was this change taken as an example from another service?  From my knowledge I don't believe any other service has added a var in their templates to replace the "insights_needs_reboot" field.